src/main/webapp/index.html

```diff
@@ -233,7 +233,7 @@ <h3 class="headline mb-0">Skills Shared By Users</h3>
             </div>
             <v-container grid-list-md text-xs-center class="Aboutus_container">
                 <v-layout align-center justify-center row fill-height>
-                    <v-flex v-for="i in members" :key="`3${i}`" xs3>
+                    <v-flex v-for="i in members" :key="`3${i.name_en}`" xs3>
```

Comment:
Why it should be `name_en`?
I didn't see any changes when I ran my localhost.

Reply:
Since i is an object, `3${i}` is actually a string with value "3[object Object]". Therefore, all keys would be the same for each member. That gives a console error that says 3 duplicate keys. Using `3${i.name_en}` will make the keys different, since then the keys would be "3Alison", "3Juno" ... and so on.